community-guidelines.md

```diff
@@ -0,0 +1,51 @@
+# Polyhack Community Guidelines
+
+**[#polyhack](https://github.com/polyhack)** is a community of
+programmers based on the IRC freenode network. It is specifically a
+polyglot group concerned to discuss programming ideas over the long-term.
+
+Its membership is predominantly from the Australian east coast but
+includes members from North America, New Zealand, and further. It was
+created in August, 2012.
+
+
+## Specific Guidelines
+
+ - Your guess is as good as mine
+
+
+
+## Bans
+
+There has been some discussion about the process for the rare case that
+someone exibits behaviour that is blatantly or consistently toxic to the
```

Comment:
Maybe, or find an unambiguous synonym. To me, toxic (in the context of a community) has a pretty clear meaning. Maybe a glossary would be handy for anything that might need clarification, but I think that is complexity that can be avoided with careful word choice.